concentratetimer/concentratetimer.py

```diff
@@ -69,13 +72,47 @@ def create_widgets(self):
 
     def get_goal(self):
         # TODO: get all goals for all clocks for the day
-        self.goal = simpledialog.askstring(title="Set your goals",
-                                           prompt="What's your goal for this clock:")
-        while not self.goal:
```

Comment:
My gut feeling shows by removing `while` will result in that ctimer does not wait for dialogue window input.

Reply:
hmmm I am probably wrong. Some more complicated logic is used in the pull request. I need more time to follow up your logic.